Sources/SwiftWarningControl/WarningControlRegions.swift

```diff
@@ -157,7 +158,8 @@ extension WarningControlRegionTree: CustomDebugStringConvertible {
       let spacing = String(repeating: "  ", count: indent)
       result += "\(spacing)[\(node.range.lowerBound), \(node.range.upperBound)]"
       if !node.warningGroupControls.isEmpty {
-        result += " id(s): \(node.warningGroupControls.keys.map { $0.identifier }.joined(separator: ", "))\n"
+        result +=
+          " control(s): \(node.warningGroupControls.map { $0.key.identifier + ":" + $0.value.rawValue }.joined(separator: ", "))\n"
```

Comment:
Nitpick, think this will look nicer with a space 😉 
```suggestion
          " control(s): \(node.warningGroupControls.map { $0.key.identifier + ": " + $0.value.rawValue }.joined(separator: ", "))\n"
```